drake/CMakeLists.txt

```diff
@@ -326,6 +322,12 @@ if (MATLAB_FOUND)
     PROPERTIES TIMEOUT 300)
 endif()
 
+option(BUILD_DOCUMENTATION "Build Doxygen and Sphinx documentation." ON)
```

Comment:
Of course, we have a ticket to build documentation on PRs. Is that ticket now invalid?